build.py

```diff
@@ -351,11 +351,18 @@ def walk( project, configs, buildDir ) :
 	help = "The filename of the tarball package to create.",
 )
 
+parser.add_argument(
+	"--jobs",
+	type = int,
+	default = multiprocessing.cpu_count(),
+	help = "The number of build jobs to run in parallel. Defaults to cpu_count."
+)
+
 args = parser.parse_args()
 
 variables = {
 	"buildDir" : args.buildDir,
-	"jobs" : multiprocessing.cpu_count(),
+	"jobs" : args.jobs,
```

Comment:
Let's do the nice thing whereby you can pass negative numbers to reserve a few cores.

Reply:
OK, since this isn't essential, but the digest fix is, I'm going to merge anyway.